[Bug] My PR does not pass some not related tests (see attached log)

Hi there, not long ago I submitted a PR with fixes for ASTextNode. Somehow it does not pass tests because of an error in -i believe- not related tests.

Building & testing AsyncDisplayKit.
Analyzing dependencies
Downloading dependencies
Using FBSnapshotTestCase (2.1.4)
Using FLAnimatedImage (1.0.12)
Using JGMethodSwizzler (2.0.1)
Using OCMock (2.2.4)
Using PINCache (3.0.1-beta.2)
Using PINRemoteImage (3.0.0-beta.7)
Generating Pods project
Integrating client project
Sending stats
[32mPod installation complete! There are 4 dependencies from the Podfile and 6 total pods installed.[0m
▸ Building AsyncDisplayKit/AsyncDisplayKit [Debug]
▸ Check Dependencies
▸ Compiling ASLayoutTransition.mm
▸ Compiling ASViewController.mm
▸ Compiling _ASCollectionReusableView.m
▸ Compiling ASTipsWindow.m
▸ Compiling ASDefaultPlaybackButton.m
▸ Compiling ASLayoutManager.m
▸ Compiling ASDimensionDeprecated.mm
▸ Compiling ASMapNode.mm
▸ Compiling AsyncDisplayKit+Tips.m
▸ Compiling ASDefaultPlayButton.m
▸ Compiling _ASAsyncTransaction.mm
▸ Compiling ASHashing.m
▸ Compiling ASStackPositionedLayout.mm
▸ Compiling _ASHierarchyChangeSet.mm
▸ Compiling ASTipProvider.m
▸ Compiling _ASCoreAnimationExtras.mm
▸ Compiling ASMainSerialQueue.mm
▸ Compiling _ASDisplayLayer.mm
▸ Compiling _ASDisplayView.mm
▸ Compiling ASVideoNode.mm
▸ Compiling ASTip.m
▸ Compiling _ASPendingState.mm
▸ Compiling ASNodeController+Beta.m
▸ Compiling ASDisplayNode+Ancestry.m
▸ Compiling ASAbstractLayoutController.mm
▸ Compiling ASTextKitContext.mm
▸ Compiling ASPagerFlowLayout.m
▸ Compiling ASElementMap.m
▸ Compiling ASTextKitFontSizeAdjuster.mm
▸ Compiling ASTextInput.m
▸ Compiling ASBackgroundLayoutSpec.mm
▸ Compiling ASDelegateProxy.m
▸ Compiling ASBasicImageDownloader.mm
▸ Compiling ASBatchContext.mm
▸ Compiling ASCellNode.mm
▸ Compiling ASCollectionLayoutContext.mm
▸ Compiling ASCenterLayoutSpec.mm
▸ Compiling ASCollectionNode.mm
▸ Compiling ASLayoutElement.mm
▸ Compiling ASVisibilityProtocols.m
▸ Compiling ASResponderChainEnumerator.m
▸ Compiling ASWeakProxy.m
▸ Compiling ASTableNode.mm
▸ Compiling _ASDisplayViewAccessiblity.mm
▸ Compiling ASCollectionView.mm
▸ Compiling ASLayerBackingTipProvider.m
▸ Compiling ASCollectionViewLayoutController.m
▸ Compiling ASControlNode.mm
▸ Compiling UIImage+ASConvenience.m
▸ Compiling ASDataController.mm
▸ Compiling AsyncDisplayKit+Debug.m
▸ Compiling ASDimension.mm
▸ Compiling ASDisplayNode+AsyncDisplay.mm
▸ Compiling ASPagerNode.m
▸ Compiling ASTextNode2.mm
▸ Compiling ASDisplayNode+DebugTiming.mm
▸ Compiling ASTextKitRenderer+Positioning.mm
▸ Compiling ASImageNode+AnimatedImage.mm
▸ Compiling ASCollectionElement.mm
▸ Compiling ASDisplayNode+UIViewBridge.mm
▸ Compiling ASCollectionLayoutState.m
▸ Compiling ASDisplayNode.mm
▸ Compiling ASDisplayNodeExtras.mm
▸ Compiling ASEditableTextNode.mm
▸ Compiling ASTextKitRenderer.mm
▸ Compiling ASWeakSet.m
▸ Compiling ASTableLayoutController.m
▸ Compiling ASHighlightOverlayLayer.mm
▸ Compiling _ASCollectionViewCell.m
▸ Compiling ASMutableElementMap.m
▸ Compiling ASImageNode+CGExtras.m
▸ Compiling ASCollectionLayout.mm
▸ Compiling ASStackUnpositionedLayout.mm
▸ Compiling ASImageNode.mm
▸ Compiling ASTextKitComponents.mm
▸ Compiling ASInsetLayoutSpec.mm
▸ Compiling ASInternalHelpers.m
▸ Compiling ASLayout.mm
▸ Compiling ASButtonNode.mm
▸ Compiling ASTextLayout.m
▸ Compiling ASTextNodeWordKerner.m
▸ Compiling ASLayoutSpec.mm
▸ Compiling ASPendingStateController.mm
▸ Compiling ASTextKitTailTruncater.mm
▸ Compiling ASObjectDescriptionHelpers.m
▸ Compiling ASMultiplexImageNode.mm
▸ Compiling ASMutableAttributedStringBuilder.m
▸ Compiling ASNetworkImageNode.mm
▸ Compiling ASOverlayLayoutSpec.mm
▸ Compiling ASTwoDimensionalArrayUtils.m
▸ Compiling ASTipsController.m
▸ Compiling ASRangeController.mm
▸ Compiling ASBatchFetching.m
▸ Compiling ASNavigationController.m
▸ Compiling ASTraceEvent.m
▸ Compiling ASRatioLayoutSpec.mm
▸ Compiling ASTextKitShadower.mm
▸ Compiling ASTextKitAttributes.mm
▸ Compiling ASDisplayNode+Yoga.mm
▸ Compiling ASRectTable.m
▸ Compiling ASDisplayNodeTipState.m
▸ Compiling ASScrollDirection.m
▸ Compiling ASScrollNode.mm
▸ Compiling ASDisplayNode+Layout.mm
▸ Compiling ASVideoPlayerNode.mm
▸ Compiling ASPageTable.m
▸ Compiling ASStackLayoutSpec.mm
▸ Compiling ASRelativeLayoutSpec.mm
▸ Compiling ASEventLog.mm
▸ Compiling ASTraitCollection.m
▸ Compiling ASWeakMap.m
▸ Compiling ASDisplayNode+Convenience.m
▸ Compiling ASCollectionFlowLayoutDelegate.m
▸ Compiling ASRunLoopQueue.mm
▸ Compiling ASTabBarController.m
▸ Compiling ASTextLine.m
▸ Compiling ASAbsoluteLayoutSpec.mm
▸ Compiling NSAttributedString+ASText.m
▸ Compiling ASDimensionInternal.mm
▸ Compiling ASCollectionViewLayoutInspector.m
▸ Compiling _ASTransitionContext.m
▸ Compiling ASTableView.mm
▸ Compiling ASTextNode.mm
▸ Compiling ASDisplayNodeLayout.mm
▸ Compiling ASPINRemoteImageDownloader.m
▸ Compiling AsyncDisplayKit+IGListKitMethods.m
▸ Compiling CoreGraphics+ASConvenience.m
▸ Compiling ASTextKitEntityAttribute.m
▸ Compiling ASPhotosFrameworkImageRequest.m
▸ Compiling ASTextKitCoreTextAdditions.m
▸ Compiling ASTextUtilities.m
▸ Compiling UIResponder+AsyncDisplayKit.m
▸ Compiling ASLayoutSpec+Subclasses.mm
▸ Compiling ASTipNode.m
▸ Compiling UICollectionViewLayout+ASConvenience.m
▸ Compiling ASCollectionViewFlowLayoutInspector.m
▸ Compiling ASTextKitRenderer+TextChecking.mm
▸ Linking AsyncDisplayKit
▸ Generating 'AsyncDisplayKit.framework.dSYM'
▸ Copying ASAvailability.h
▸ Copying ASDisplayNodeInternal.h
▸ Copying ASDisplayNode+Beta.h
▸ Touching AsyncDisplayKit.framework
▸ Signing /Users/buildkite/Library/Developer/Xcode/DerivedData/AsyncDisplayKit-dgabgdhhjbwevtaicutqvkenbyal/Build/Products/Debug-iphonesimulator/AsyncDisplayKit.framework
▸ Building AsyncDisplayKit/AsyncDisplayKitTestHost [Debug]
▸ Check Dependencies
▸ Compiling AppDelegate.m
▸ Linking AsyncDisplayKitTestHost
▸ Generating 'AsyncDisplayKitTestHost.app.dSYM'
▸ Touching AsyncDisplayKitTestHost.app
▸ Signing /Users/buildkite/Library/Developer/Xcode/DerivedData/AsyncDisplayKit-dgabgdhhjbwevtaicutqvkenbyal/Build/Products/Debug-iphonesimulator/AsyncDisplayKitTestHost.app
▸ Building Pods/FBSnapshotTestCase [Debug]
▸ Check Dependencies
▸ Compiling FBSnapshotTestCase-dummy.m
▸ Compiling FBSnapshotTestCase.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/FBSnapshotTestCase/FBSnapshotTestCase/FBSnapshotTestController.h:92:9: parameter 'error' not found in the function declaration [-Wdocumentation]

 @param error An error to log in an XCTAssert() macro if the method fails (missing reference image, images differ, etc).
^



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/FBSnapshotTestCase/FBSnapshotTestCase/FBSnapshotTestController.h:105:9: parameter 'error' not found in the function declaration [-Wdocumentation]

 @param error An error to log in an XCTAssert() macro if the method fails (missing reference image, images differ, etc).
        ^~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/FBSnapshotTestCase/FBSnapshotTestCase/FBSnapshotTestController.h:115:9: parameter 'view' not found in the function declaration [-Wdocumentation]

 @param view The view or layer to snapshot.
        ^~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/FBSnapshotTestCase/FBSnapshotTestCase/FBSnapshotTestController.h:119:9: parameter 'error' not found in the function declaration [-Wdocumentation]

 @param error An error to log in an XCTAssert() macro if the method fails (missing reference image, images differ, etc).
        ^~~~


▸ Compiling FBSnapshotTestCasePlatform.m
▸ Compiling FBSnapshotTestController.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/FBSnapshotTestCase/FBSnapshotTestCase/FBSnapshotTestController.h:92:9: parameter 'error' not found in the function declaration [-Wdocumentation]

 @param error An error to log in an XCTAssert() macro if the method fails (missing reference image, images differ, etc).
        ^~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/FBSnapshotTestCase/FBSnapshotTestCase/FBSnapshotTestController.h:105:9: parameter 'error' not found in the function declaration [-Wdocumentation]

 @param error An error to log in an XCTAssert() macro if the method fails (missing reference image, images differ, etc).
        ^~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/FBSnapshotTestCase/FBSnapshotTestCase/FBSnapshotTestController.h:115:9: parameter 'view' not found in the function declaration [-Wdocumentation]

 @param view The view or layer to snapshot.
        ^~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/FBSnapshotTestCase/FBSnapshotTestCase/FBSnapshotTestController.h:119:9: parameter 'error' not found in the function declaration [-Wdocumentation]

 @param error An error to log in an XCTAssert() macro if the method fails (missing reference image, images differ, etc).
        ^~~~


▸ Compiling UIApplication+StrictKeyWindow.m
▸ Compiling UIImage+Compare.m
▸ Compiling UIImage+Diff.m
▸ Compiling UIImage+Snapshot.m
▸ Building library libFBSnapshotTestCase.a
▸ Building Pods/FLAnimatedImage [Debug]
▸ Check Dependencies
▸ Compiling FLAnimatedImage-dummy.m
▸ Compiling FLAnimatedImage.m
▸ Compiling FLAnimatedImageView.m
▸ Building library libFLAnimatedImage.a
▸ Building Pods/JGMethodSwizzler [Debug]
▸ Check Dependencies
▸ Compiling JGMethodSwizzler-dummy.m
▸ Compiling JGMethodSwizzler.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/JGMethodSwizzler/JGMethodSwizzler/JGMethodSwizzler.h:77:9: parameter 'replacement' not found in the function declaration [-Wdocumentation]

 @param replacement The replacement block to use for swizzling the method. Its signature needs to be: return_type ^(id self, ...).
        ^~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/JGMethodSwizzler/JGMethodSwizzler/JGMethodSwizzler.h:88:9: parameter 'replacement' not found in the function declaration [-Wdocumentation]

 @param replacement The replacement block to use for swizzling the method. Its signature needs to be: return_type ^(id self, ...).
        ^~~~~~~~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/JGMethodSwizzler/JGMethodSwizzler/JGMethodSwizzler.h:184:9: parameter 'replacement' not found in the function declaration [-Wdocumentation]

 @param replacement The replacement block to use for swizzling the method. Its signature needs to be: return_type ^(id self, ...).
        ^~~~~~~~~~~


▸ Building library libJGMethodSwizzler.a
▸ Building Pods/OCMock [Debug]
▸ Check Dependencies
▸ Compiling NSInvocation+OCMAdditions.m
▸ Compiling NSMethodSignature+OCMAdditions.m
▸ Compiling NSNotificationCenter+OCMAdditions.m
▸ Compiling NSObject+OCMAdditions.m
▸ Compiling OCClassMockObject.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/OCMock/Source/OCMock/OCClassMockObject.m:203:3: '@interface' command should not be used in a comment attached to a non-interface declaration [-Wdocumentation]

 @interface NSObject (__NSIsKinds)
        ^~~~~~~~~~~


▸ Compiling OCMArg.m
▸ Compiling OCMBlockCaller.m
▸ Compiling OCMBoxedReturnValueProvider.m
▸ Compiling OCMConstraint.m
▸ Compiling OCMExceptionReturnValueProvider.m
▸ Compiling OCMIndirectReturnValueProvider.m
▸ Compiling OCMNotificationPoster.m
▸ Compiling OCMObserverRecorder.m
▸ Compiling OCMock-dummy.m
▸ Compiling OCMockObject.m
▸ Compiling OCMockRecorder.m
▸ Compiling OCMPassByRefSetter.m
▸ Compiling OCMRealObjectForwarder.m
▸ Compiling OCMReturnValueProvider.m
▸ Compiling OCObserverMockObject.m
▸ Compiling OCPartialMockObject.m
▸ Compiling OCPartialMockRecorder.m
▸ Compiling OCProtocolMockObject.m
▸ Building library libOCMock.a
▸ Building Pods/PINCache [Debug]
▸ Check Dependencies
▸ Compiling PINCache-dummy.m
▸ Compiling PINCache.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINCache/PINCache/PINCacheObjectSubscripting.h:26:9: parameter 'object' not found in the function declaration [-Wdocumentation]

 @param object An object to be assigned for the key.
 ~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


▸ Compiling PINDiskCache.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINCache/PINCache/PINCacheObjectSubscripting.h:26:9: parameter 'object' not found in the function declaration [-Wdocumentation]

 @param object An object to be assigned for the key.
        ^~~~~~


▸ Compiling PINMemoryCache.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINCache/PINCache/PINCacheObjectSubscripting.h:26:9: parameter 'object' not found in the function declaration [-Wdocumentation]

 @param object An object to be assigned for the key.
        ^~~~~~


▸ Compiling PINOperationGroup.m
▸ Compiling PINOperationQueue.m
▸ Building library libPINCache.a
▸ Building Pods/PINRemoteImage [Debug]
▸ Check Dependencies
▸ Compiling FLAnimatedImageView+PINRemoteImage.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:146:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:155:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~


▸ Compiling NSData+ImageDetectors.m
▸ Compiling PINAlternateRepresentationProvider.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:146:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:155:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~


▸ Compiling PINAnimatedImage.m
▸ Compiling PINAnimatedImageManager.m
▸ Compiling PINButton+PINRemoteImage.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:146:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:155:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~


▸ Compiling PINCache+PINRemoteImageCaching.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINCache/PINCache/PINCacheObjectSubscripting.h:26:9: parameter 'object' not found in the function declaration [-Wdocumentation]

 @param object An object to be assigned for the key.
        ^~~~~~~~~~~~~~~~~~~~~~


▸ Compiling PINDataTaskOperation.m
▸ Compiling PINImage+DecodedImage.m
▸ Compiling PINImage+WebP.m
▸ Compiling PINImageView+PINRemoteImage.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:146:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:155:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~


▸ Compiling PINProgressiveImage.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:146:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:155:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~


▸ Compiling PINRemoteImage-dummy.m
▸ Compiling PINRemoteImageBasicCache.m
▸ Compiling PINRemoteImageCallbacks.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:146:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:155:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~


▸ Compiling PINRemoteImageCategoryManager.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:146:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:155:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~


▸ Compiling PINRemoteImageDownloadTask.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:146:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:155:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~


▸ Compiling PINRemoteImageManager.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:146:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:155:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINCache/PINCache/PINCacheObjectSubscripting.h:26:9: parameter 'object' not found in the function declaration [-Wdocumentation]

 @param object An object to be assigned for the key.
        ^~~~~~~~~~~~~~~~~~~~~~


▸ Compiling PINRemoteImageManagerResult.m
▸ Compiling PINRemoteImageMemoryContainer.m
▸ Compiling PINRemoteImageProcessorTask.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:146:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:155:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~


▸ Compiling PINRemoteImageTask.m

⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:146:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~



⚠️  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Pods/PINRemoteImage/Pod/Classes/PINRemoteImageManager.h:155:9: parameter 'alternativeRepDelegate' not found in the function declaration [-Wdocumentation]

 @param alternativeRepDelegate a delegate which conforms to the PINRemoteImageManagerAlternateRepresentationProvider protocol. Provide a delegate if you want to have non image results. @see PINRemoteImageManagerAlternateRepresentationProvider for an example.
        ^~~~~~~~~~~~~~~~~~~~~~


▸ Compiling PINRemoteLock.m
▸ Compiling PINURLSessionManager.m
▸ Building library libPINRemoteImage.a
▸ Building Pods/Pods-AsyncDisplayKitTests [Debug]
▸ Check Dependencies
▸ Compiling Pods-AsyncDisplayKitTests-dummy.m
▸ Building library libPods-AsyncDisplayKitTests.a
▸ Building AsyncDisplayKit/AsyncDisplayKitTests [Debug]
▸ Check Dependencies
▸ Running script '[CP] Check Pods Manifest.lock'
▸ Compiling ASBasicImageDownloaderContextTests.m

❌  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Tests/ASBasicImageDownloaderContextTests.m:18:9: could not build module 'AsyncDisplayKit'

#import <AsyncDisplayKit/ASBasicImageDownloader.h>
^


▸ Compiling ASBasicImageDownloaderTests.m
** TEST BUILD FAILED **


The following build commands failed:
	CompileC /Users/buildkite/Library/Developer/Xcode/DerivedData/AsyncDisplayKit-dgabgdhhjbwevtaicutqvkenbyal/Build/Intermediates/AsyncDisplayKit.build/Debug-iphonesimulator/AsyncDisplayKitTests.build/Objects-normal/x86_64/ASBasicImageDownloaderContextTests.o Tests/ASBasicImageDownloaderContextTests.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
2017-05-20 13:29:17.189 xcodebuild[66035:3701680]  IDETestOperationsObserverDebug: Writing diagnostic log for test session to:
/Users/buildkite/Library/Developer/Xcode/DerivedData/AsyncDisplayKit-dgabgdhhjbwevtaicutqvkenbyal/Logs/Test/1392CB6D-96AE-43D3-B73C-0AF787382478/Session-AsyncDisplayKitTests-2017-05-20_132917-AwQJyA.log
2017-05-20 13:29:17.189 xcodebuild[66035:3701651] [MT] IDETestOperationsObserverDebug: (FBA9F3C2-0FCA-421A-9249-C3908BE8A869) Beginning test session AsyncDisplayKitTests-FBA9F3C2-0FCA-421A-9249-C3908BE8A869 at 2017-05-20 13:29:17.188 with Xcode 8C1002 on target <DVTiPhoneSimulator: 0x7ff19170eae0> {
		SimDevice: SimDevice : iPhone 7 (47ACC27E-E45C-42CE-8146-27256C3CC33B) : state={ Booted } deviceType={ SimDeviceType : com.apple.CoreSimulator.SimDeviceType.iPhone-7 } runtime={ SimRuntime : 10.2 (14C89) - com.apple.CoreSimulator.SimRuntime.iOS-10-2 }
} (10.2 (14C89))
▸ Building AsyncDisplayKit/AsyncDisplayKit [Debug]
▸ Check Dependencies
▸ Building AsyncDisplayKit/AsyncDisplayKitTestHost [Debug]
▸ Check Dependencies
▸ Linking AsyncDisplayKitTestHost
▸ Generating 'AsyncDisplayKitTestHost.app.dSYM'
▸ Touching AsyncDisplayKitTestHost.app
▸ Signing /Users/buildkite/Library/Developer/Xcode/DerivedData/AsyncDisplayKit-dgabgdhhjbwevtaicutqvkenbyal/Build/Products/Debug-iphonesimulator/AsyncDisplayKitTestHost.app
▸ Building Pods/FBSnapshotTestCase [Debug]
▸ Check Dependencies
▸ Building Pods/FLAnimatedImage [Debug]
▸ Check Dependencies
▸ Building Pods/JGMethodSwizzler [Debug]
▸ Check Dependencies
▸ Building Pods/OCMock [Debug]
▸ Check Dependencies
▸ Building Pods/PINCache [Debug]
▸ Check Dependencies
▸ Building Pods/PINRemoteImage [Debug]
▸ Check Dependencies
▸ Building Pods/Pods-AsyncDisplayKitTests [Debug]
▸ Check Dependencies
▸ Building AsyncDisplayKit/AsyncDisplayKitTests [Debug]
▸ Check Dependencies
▸ Running script '[CP] Check Pods Manifest.lock'
▸ Compiling ASBasicImageDownloaderContextTests.m

❌  /Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Tests/ASBasicImageDownloaderContextTests.m:18:9: could not build module 'AsyncDisplayKit'

#import <AsyncDisplayKit/ASBasicImageDownloader.h>
^


▸ Compiling ASBasicImageDownloaderTests.m
2017-05-20 13:29:17.849 xcodebuild[66035:3701651] Error Domain=IDETestOperationsObserverErrorDomain Code=3 "Test operation was canceled. If you believe this error represents a bug, please attach the log file at /Users/buildkite/Library/Developer/Xcode/DerivedData/AsyncDisplayKit-dgabgdhhjbwevtaicutqvkenbyal/Logs/Test/1392CB6D-96AE-43D3-B73C-0AF787382478/Session-AsyncDisplayKitTests-2017-05-20_132917-AwQJyA.log" UserInfo={NSLocalizedDescription=Test operation was canceled. If you believe this error represents a bug, please attach the log file at /Users/buildkite/Library/Developer/Xcode/DerivedData/AsyncDisplayKit-dgabgdhhjbwevtaicutqvkenbyal/Logs/Test/1392CB6D-96AE-43D3-B73C-0AF787382478/Session-AsyncDisplayKitTests-2017-05-20_132917-AwQJyA.log}
2017-05-20 13:29:17.850 xcodebuild[66035:3701743] Connection peer refused channel request for "dtxproxy:XCTestManager_IDEInterface:XCTestManager_DaemonConnectionInterface"; channel canceled <DTXChannel: 0x7ff192409aa0>

Testing failed:
	Umbrella header for module 'AsyncDisplayKit' does not include header '/Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Source/AsyncDisplayKit+IGListKitMethods.h'
	Umbrella header for module 'AsyncDisplayKit' does not include header '/Users/Shared/buildkite/builds/iosf-garrett-VMgI1sNHYyXB.dyn.pinadmin.com-1/pinterest/texture/Source/IGListAdapter+AsyncDisplayKit.h'
	Could not build module 'AsyncDisplayKit'
** TEST FAILED **


The following build commands failed:
	CompileC /Users/buildkite/Library/Developer/Xcode/DerivedData/AsyncDisplayKit-dgabgdhhjbwevtaicutqvkenbyal/Build/Intermediates/AsyncDisplayKit.build/Debug-iphonesimulator/AsyncDisplayKitTests.build/Objects-normal/x86_64/ASBasicImageDownloaderContextTests.o Tests/ASBasicImageDownloaderContextTests.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)


Oh no! You walked directly into the slavering fangs of a lurking grue!
**** You have died ****

--- a/Source/AsyncDisplayKit.h
+++ b/Source/AsyncDisplayKit.h
@@ -55,11 +55,6 @@
 #import <AsyncDisplayKit/ASSectionController.h>
 #import <AsyncDisplayKit/ASSupplementaryNodeSource.h>
 
-#if AS_IG_LIST_KIT
-#import <AsyncDisplayKit/IGListAdapter+AsyncDisplayKit.h>
-#import <AsyncDisplayKit/AsyncDisplayKit+IGListKitMethods.h>
-#endif
-
 #import <AsyncDisplayKit/ASScrollNode.h>
 
 #import <AsyncDisplayKit/ASPagerFlowLayout.h>
@@ -128,3 +123,6 @@
 #import <AsyncDisplayKit/ASDisplayNode+Deprecated.h>
 
 #import <AsyncDisplayKit/ASCollectionNode+Beta.h>
+
+#import <AsyncDisplayKit/IGListAdapter+AsyncDisplayKit.h>
+#import <AsyncDisplayKit/AsyncDisplayKit+IGListKitMethods.h>

